.env

```diff
@@ -51,7 +51,5 @@ COPY_CONCURRENCY=10
 
 # Variables for generate tiles using PGquery
 PGHOSTS_LIST=
-# Uncomment when used PGquery
-#EXPORT_DIR=/import
```

Comment:
Please leave these comments so that this patch is easier to rebase onto future changes from OMT/MapTiler

Reply:
This change came from the original MapTiler repo. I can readd these comments

Reply:
Oh, OK, I didn't realize what was going on. Please don't re-add the comments, instead give me a minute to figure out the discrepancy.